index.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.menuImg');
+const menuList = document.querySelector('.menu_items');
+const closeHamburger = document.querySelector('#btn_close');
+const logoName = document.querySelector('.logo-name');
+
+hamburger.addEventListener('click', () => {
+  if (menuList.style.display !== 'block') {
+    menuList.style.display = 'block';
+    logoName.style.display = 'none';
+    hamburger.style.display = 'none';
+  }
+});
+
+closeHamburger.addEventListener('click', () => {
```

Comment:
[ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear. your menu is still displayed. kindly fix that.🤓

Reply:
- [ ] Just as the previous reviewer requested, please when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear, please fix it see the video below ⬇️ 


https://user-images.githubusercontent.com/69638013/208862795-23545b9f-4305-4bb7-b406-0439158682bb.mp4